x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/pagerduty/pagerduty_params.tsx

```diff
@@ -194,6 +200,7 @@ const PagerDutyParamsFields: React.FunctionComponent<ActionParamsProps<PagerDuty
               messageVariables={messageVariables}
               paramsProperty={'dedupKey'}
               inputTargetValue={dedupKey}
+              defaultValue={`{{${AlertProvidedActionVariables.alertInstanceId}}}`}
```

Comment:
Should these be something like the following to avoid interference between alerts? (ex: 2 index threshold alerts using `all documents` as the `alertInstanceId` would interference using the same PagerDuty incident).

```suggestion
              defaultValue={`{{${AlertProvidedActionVariables.alertId}}}:{{${AlertProvidedActionVariables.alertInstanceId}}}`}
```

Reply:
Good catch! 👍